wagtail_footnotes/test/fixtures/core.json

```diff
@@ -0,0 +1,104 @@
+[
+{
+  "model": "wagtailcore.page",
```

Comment:
question: any reason not to use wagtail-factories?

Reply:
I'm not that familiar with the package and I wanted to not add learning time here but I'd be happy to dig into it if you think it's worth using.

Reply:
perhaps one for the future. well worth it though in that you don't have to keep updating fixtures

Reply:
Ah those fixtures aren't used in tests, they were a convenience when I was figuring the testing out but not needed now, I think. Will look at them again to be sure though.

Reply:
ah, in that case they are best removed :)

Reply:
Fixtures are now gone.